### ISSUE TITLE
Change background color from #E0F3F8 to #E1F4F9 

### ISSUE THREAD

**Issue by femmebot** [18F/nsf-sbir#553]
Based on accessibility audit ([see Slack convo](https://gsa-tts.slack.com/archives/C2TH5SAT0/p1498251698288806))

**Comment by gemfarmer** [comment]
See #555, a PR that makes this change

**Comment by OpenGlobe** [comment]
Closing this issue - PR merged.
